src/main/java/com/google/sps/data/Note.java

```diff
@@ -1,16 +1,135 @@
 package com.google.sps.data;
 
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageSource;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.Symbol;
+import com.google.cloud.vision.v1.Word;
+import com.google.cloud.vision.v1.Paragraph;
+import com.google.cloud.vision.v1.Block;
+import com.google.cloud.vision.v1.Page;
+import com.google.cloud.vision.v1.TextAnnotation;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+
 public final class Note {
+    private long id;
+    private final String imageUrl;
+    private final String message;
+
+    // how I'd imagine the NLP text can be sorted
+    // key is the keyword/heading
+    // value is a list of related sentences
+    private HashMap<String, List<String>> categorizedText;
```

Comment:
Should we tie this to a session key? How do we differentiate different users using the platform?

Reply:
I added the session key to the `Note Entity` in Datastore and held off on adding it to the `Note` class for now as there wasn't really a need for it in the class.  Should I replace the current `id` in this class with that id instead?

Reply:
Yeah that can work!

Reply:
If we are only making this for one user however, the way we have it should be fine though